motionplan/motionPlanner.go

```diff
@@ -61,44 +59,6 @@ func PlanMotion(ctx context.Context,
 	)
 }
 
-// PlanRobotMotion plans a motion to destination for a given frame. A robot object is passed in and current position inputs are determined.
```

Comment:
This was causing an issue with a circular dependency for a while and I deleted it with the intention of adding it back in.

Reply:
Ok so I remember now why I removed it.  Its no longer possible to get the frame system from a robot (and it shouldn't be just for the purposes of a helper function only accessible in the GoSDK).  

My feeling is that if we need a helper function at this level it should live attached to a motion service where it will have access to the frame system service as a dependency.  Then any time you are developing in the GoSDK. you will have access to the builtin motion client which you can use to call it

Reply:
It creates a circular dependency between the robot and the framesystem

Reply:
Robot's been removed from the framesystem package import list now

Reply:
The thing that could be done to solve this would be writing some new robot level helper function to expose its internal `framesystem`.  But this feels like something that we should avoid doing because I think it breaks some of the abstractions around how internal services work.  

Is there a problem with the workaround solution I outlined earlier? `ms.PlanRobotMotion` would be much easier to implement

Reply:
> Robot's been removed from the framesystem package import list now

Right but if you add `framesystem.FromRobot` it is added back

Reply:
This seems like a helpful method for the Go SDK. Can we move it somewhere else to avoid the circular dependency?

Reply:
I investigated the fix to this and its bigger than I want to include in this PR and will require motion team approval.  RSDK-3036 now exists to track this, so I would like to remove this after all

Reply:
RSDK-3036's description is "Stoppable motionService.Move"; would that allow you to implement this function elsewhere? I'm fine deferring to the motion team on whether or not this method should be removed; I think there are probably few consumers of this API beyond you all for the time being.

Reply:
Ah sorry I linked the wrong ticket its RSDK-3039